domains/brandnewwick.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "alephdfilms",
+        "email": "alephdiallo@mail.com"
+    },
+    "record": {
+        "A": ["185.199.108.153"]
```

Comment:
Missing 3 other A records. You need the other 3 to work properly.

Reply:
> Missing 3 other A records. You need the other 3 to work properly.

Just added them...